src/components/layout/OrganizeLayout.tsx

```diff
@@ -36,7 +36,7 @@ const OrganizeLayout: FunctionComponent<OrganizeLayoutProps> = ({ children, orgI
                         <BreadcrumbTrail/>
                     </Box>
                     <Box display="flex" justifyContent="end" position="absolute" right={ 0 } top={ 0 } width={ 0.5 } zIndex={ 10000 }>
-                        <SearchDrawer />
+                        <SearchDrawer { ...{ orgId } } />
```

Comment:
Why this syntax over `orgId={ orgId }`?

Reply:
TBH I don't love passing in the`orgId` down through all children. Sometimes when something is getting passed through children like this, I use this syntax. Then if we need to pass more things like this, we just add them to the list. But I was thinking maybe we want to move this to a context eventually...

Reply:
I was thinking about this, passing the props through the layout. I was thinking it would possibly be better to use a context here, because so many of the children depend on the `orgId`. Curious what you think.

However, I will change the syntax, I just did that at the time, not knowing how many props I would pass in.

Reply:
If you want to switch to using context for this I wouldn't mind. :)

The "active" organization is derived from the URL, so it will be global for any page and IMO is a good candidate for context data.

Reply:
Another place we could use context for orgId is in the calendar bar component for the links to the campaigns and events

Reply:
yes i think moving over to a context so that any page with an org is a good itea, to avoid having to pass the org id down thru the component tree. i will look in to that, and probably do it as a new PR after the campaign PR is merged, then can migrate all use of it in that PR.

Reply:
In the meantime, is it OK I leave it and then move to a context soon?